source/pages/Transactions/Transaction.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+
+import { Layout } from 'components/index';
+import { useQueryData } from 'hooks/index';
+import { getController } from 'utils/browser';
+
+import Fee from './Fee';
+import TransactionConfirmation from './TransactionConfirmation';
+
+const titleResolver = (txType: string) => {
+  switch (txType) {
```

Comment:
use `camelCaseToText` to validate tx type instead of switch case, as well as in callbackResolver

Reply:
\`camelCaseToText\` doesn't work properly with "NFT" (splits all letters)